reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/model/FiscalEventResponse.java

```diff
@@ -0,0 +1,41 @@
+package org.egov.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.egov.common.contract.response.ResponseHeader;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains the ResponseHeader and the enriched fiscal information
+ */
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2021-08-09T17:28:42.515+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class FiscalEventResponse {
+    
+    @JsonProperty("responseInfo")
+    private ResponseHeader responseInfo = null;
```

Comment:
Isn't the name of the parameter "responseHeader"?

Reply:
Yes..i copied it from mgramseva-ifix adapter. I'll change it to "responseHeader".